clients/elrond/mock_test.go

```diff
@@ -0,0 +1,28 @@
+package elrond
+
+import (
+	"context"
+
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/builders"
+)
+
+type txHandlerStub struct {
```

Comment:
why did you implement this stub here and not in testsCommon ?

Reply:
because the interface it implements is private and has private methods. It was just a way to split the code among multiple structs/files. Can not be moved unless the methods become exported which is not intended

Reply:
changed